client/components/search/index.jsx

```diff
@@ -38,7 +38,8 @@ module.exports = React.createClass( {
 		onKeyDown: React.PropTypes.func,
 		disableAutocorrect: React.PropTypes.bool,
 		onBlur: React.PropTypes.func,
-		searching: React.PropTypes.bool
+		searching: React.PropTypes.bool,
+		forceCloseButton: React.PropTypes.bool
```

Comment:
So it seems that `is-open` is a state, and not a prop — which means we can't set it.